config/settings/test.py

```diff
@@ -77,7 +77,7 @@
 SALESFORCE_USERNAME = "test@example.com"
 SALESFORCE_ARTICLE_AUTHOR_FIELD = "ArticleAuthor__c"
 SALESFORCE_ARTICLE_AUTHOR_OVERRIDE_FIELD = "ArticleAuthorOverride__c"
-SALESFORCE_ARTICLE_TYPE = "Knowledge__kav"
+SALESFORCE_ARTICLE_TYPE = "Resource__kav"
```

Comment:
@prescod I'm still curious about this.

Reply:
It was a benign error in test.py that I corrected. It was benign because test.py never actually calls into Salesforce so it could call Salesforce columns anything it likes without having any noticeable result. But having the error could obviously confuse future programmers so I fixed it. Now test.py has the same configuration as env.example and all of the real configurations.